docker-for-mac/index.md

```diff
@@ -297,15 +297,23 @@ choose to discard or not apply changes when asked.
 
 ### Kubernetes
 
-**Kubernetes is only available if you are part of the private beta for Docker for Mac 17.12. To access beta builds, you must be signed in within Docker for Mac using your Docker ID.**
+**Kubernetes is only available in the private beta of Docker for Mac 17.12 CE Edge through the [Docker Beta program](https://beta.docker.com/). To access beta builds, you must be signed in with your Docker ID within Docker for Mac: select ![whale menu](/docker-for-mac/images/whale-x.png){: .inline} -> Sign in / Create Docker ID from the menu bar.**
 
 If you are participating in the Docker beta program, you can access Docker for
 Mac 17.12 CE Edge, which includes a standalone Kubernetes server that runs on
 your Mac, so that you can test deploying your Docker workloads on Kubernetes.
+
 The Kubernetes client command, `kubectl`, is included and configured to connect
-to the local Kubernetes server.
+to the local Kubernetes server. If you have `kubectl` already installed and
+pointing to some other environment, such as minikube or a gke cluster, be sure
+to change context so that kubectl is pointing to `docker-for-desktop`:
+
+```
+kubectl config get-contexts
+kubectl config use-context docker-for-desktop
+```
 
-To log in with your Docker ID, select ![whale menu](/docker-for-mac/images/whale-x.png){: .inline} -> **Sign in / Create Docker ID** from the menu bar.
+If you installed `kubectl` with homebrew and encounter a symlink error, remove `/usr/local/bin/kubectl`.
```

Comment:
Check branding on Homebrew. I think maybe they prefer to capitalize it. There are other ways to install stuff on Mac, such as Mac Ports. So maybe make this a bit more generic.